addons/mail/static/src/discuss/core/common/discuss_notification_settings.xml

```diff
@@ -30,7 +30,7 @@
                 <h5>Channel notification settings</h5>
                 <span class="mb-1">This setting will be applied to all channels using the default category.</span>
                 <t t-foreach="store.settings.NOTIFICATIONS" t-as="item" t-key="item.id">
-                    <button class="btn d-flex my-1" t-att-class="{'bg-300' : item.id === store.settings.channel_notifications}" t-on-click="()=> this.onChangeCustomNotifications(item.id)">
+                    <button class="btn d-flex my-1" t-att-class="{'bg-300' : item.id === store.settings.channel_notifications}" t-on-click="()=> this.store.settings.setCustomNotifications(item.id)">
```

Comment:
Slightly out-of-scope of PR, and nitpicking, but would feel less obscure if `item` was renamed `notif`.
Also `id` is decent when dealing with models, but that's not the case here. Either should be turned into a model, or prefer a more relevant naming like `label`.